WebHostLib/customserver.py

```diff
@@ -133,8 +135,36 @@ def get_save(self) -> dict:
         return d
 
 
-def get_random_port():
-    return random.randint(49152, 65535)
+def get_random_port(game_ports):
+    config_range_list = game_ports.split(",")
+    available_ports = []
+    for item in config_range_list:
+        if '-' in item:
+            start, end = map(int, item.split('-'))
+            available_ports.extend(range(start, end+1))
```

Comment:
I feel like there's probably a more perfomant way to do this than dumping 15000 ints in a list every time we want to start a room....

My initial idea would be to determine how many total possible ports were possible with the provided port range(s), then just generate a random int and arithmetically mapping it a port (for example with default ports, randomly generate an int between 0 and 16383 and then add it to 49152.

My second idea would be to just do `random.randint` and say only one range is permitted in GAME_PORTS as that'd be the typical use-case anyway.

Reply:
I wouldn't disagree, but I did think about that.
One operation per server didn't seem to terrible a trade-off. 
I did run a few tests for how fast the code ran and I got on average of 0.00031/0.00032 seconds for looping 50,000 times over it. So i'm not unhappy with the speed it's operating at.

Reply:
I'm also not entirely against just forcing one range to be defined in the config, I may have gone overkill with unnecessary customisability.